app/controllers/concerns/with_user_params.rb

```diff
@@ -6,6 +6,6 @@ def user_params
   end
 
   def permissible_params
-    User.profile_fields
+    User.profile_fields + [:avatar_id, :image_url]
```

Comment:
I haven't tested it locally, but wouldn't be enough just letting the users set the avatar_id?
If they select the image_url one, just by setting the avatar_id to nil wouldn't that do it?

I'm worried that letting them set the `image_url` could lead to some undesired scenarios :confused:.

Reply:
Yes, this is a very nice observation. I mindlessly added image-url, but it's not needed. I've also added `avatar_id` on `users_controller` so it doesn't interfere with anything else, I hadn't thought of it.